android/build.gradle

```diff
@@ -63,7 +63,7 @@ android {
     compileSdkVersion 31
 
     defaultConfig {
-        minSdkVersion rootProject.ext.safeExtGet("minSdkVersion", 21)
+        minSdkVersion rootProject.ext.has("minSdkVersion") ? rootProject.ext.minSdkVersion :  21
```

Comment:
This was changed in [this PR](https://github.com/plaid/react-native-plaid-link-sdk/pull/678), but it was causing compile errors for me so I modified it.